docs/source/install-conda-build.rst

```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
 * update conda and conda-build
 
 
+.. _-conda-build-wow:
+
+Way of working
+==============
+For proper functioning, it is strongly recommended to install conda-build in
+the conda ``base`` environment. Not doing so may lead to problems.
+
+**Explanation**
+
+With earlier conda / conda-build versions it was possible to build packages in
+your own defined environment, e.g. ``my_build_env``. This was partly driven by
+the general conda recommendation not to use the ``base`` env for normal work,
+see `Conda Managing Environments`_ for instance. However conda-build is better
+viewed as part of the conda infrastructure, and not as a normal package. Hence
+installing it in the ``base`` env makes more sense. More information:
+`Must conda-build be installed in the base envt?`_
+
+**Other considerations**
+
+* Installing or updating conda-build (in fact any package) in the ``base``
+  environment needs to be run from an account with the proper permissions.
+  I.e. the same permissions as were used to install conda and the base env in
+  the first place via the Miniconda or Anaconda installers. E.g. on Windows
+  that might mean an account with administrator privileges.
+
+* `conda-verfiy`_ can be usefully added to the base environment also. This
+  will remove some warnings generated when conda-build runs. And the package
+  is useful in its own right anyway.
```

Comment:
```suggestion
* `conda-verfiy`_ is a useful package that can also be added to the base environment in order
  to remove some warnings generated when conda-build runs.
```
That last sentence is fine if you want keep it. I just figured it was redundant to saying "useful/usefully" in the first sentence.

Reply:
ok